README.md

```diff
@@ -1,5 +1,66 @@
 # docker-symfony
 
-This repository contains all the code explained in this YouTube playlist 
+This repository contains all the code explained in this [**YouTube playlist**](https://www.youtube.com/playlist?list=PLWpsZlKx38t9SEQu6_AbzBoHeQbApIcjJ)
+
+### Scripts
+*Using **Makefile***: https://makefiletutorial.com/
+
+Installs composer dependencies
+
+```shell
+make prepare
+```
+
+Build the containers
+
+```shell
+make build
+```
+
+Start the containers
+
+```shell
+make run
+```
+
+Stop the containers
+
+```shell
+make stop
+```
+
+Restart the containers
+
+```shell
+make restart
+```
+
+### Steps to follow
+
+Init the bash shell into the **Backend container**
+
+```shell
+make ssh-be
+```
+
+Update packages
+
+```shell
+composer update
+```
+
+Install packages
+
+```shell
+composer install
+```
+
+
+### Additional
```

Comment:
Maybe "Optional" would be a better word as you can use also `docker` command to check stuff :)

Reply:
Yes, you are have the reason, I've correct this part